task-sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -946,16 +964,11 @@ def _monitor_subprocess(self):
             # This listens for activity (e.g., subprocess output) on registered file objects
             alive = self._service_subprocess(max_wait_time=max_wait_time) is None
 
-            if self._exit_code is not None and self._num_open_sockets > 0:
+            if self._exit_code is not None and self._open_sockets:
                 if (
                     self._process_exit_monotonic
                     and time.monotonic() - self._process_exit_monotonic > SOCKET_CLEANUP_TIMEOUT
                 ):
-                    log.debug(
-                        "Forcefully closing remaining sockets",
-                        open_sockets=self._num_open_sockets,
-                        pid=self.pid,
-                    )
```

Comment:
This debug log added nothing -- we'd get a log message at warning from within cleanup_open_sockets if it did anything